applications/ecallmgr/src/ecallmgr_call_command.erl

```diff
@@ -767,13 +768,18 @@ get_call_pickup_app(Node, UUID, JObj, Target, Command) ->
                     ,{<<"Fetch-ID">>, wh_util:rand_hex_binary(4)}
                     | wh_api:default_headers(?APP_NAME, ?APP_VERSION)
                    ],
-    wh_amqp_worker:cast(ControlUsurp
-                        ,fun(C) -> wapi_call:publish_usurp_control(Target, C) end
-                       ),
-    lager:debug("published control usurp for ~s", [Target]),
+
+    wh_json:is_true(<<"Publish-Usurp">>, JObj, 'true')
```

Comment:
For readability I would prefer to use a case-clause here.  It would also allow you to log when the usurp is supressed.